scripts/jenkins/Jenkinsfile

```diff
@@ -7,7 +7,7 @@ pipeline {
     environment {
         SYS_URL = 'UNKOWN'
         SYS_VER = 'UNKOWN'
-        BUILD_VERBOSE = 'no'
+        BUILD_VERBOSE = 'yes'
```

Comment:
This will eventually need to be reverted either right before merging into `develop` or soon there after.

Reply:
My initial PR never finished compiling, sat on 'Build' for 17 hours. I tried it multiple times. So I turned this on and then it finished building without any other changes. :/
Yes, I sure hope I can turn it off and it will still build when everything is done.

Reply:
Reverted